### PR TITLE
E2E Tests to validate postinstall configurations of OpenSearch component

### PR DESCRIPTION
**Pull request by anajosep**
# Description

E2E Tests to validate the various settings of OpenSearch component, that could be modified post install of the Verrazzano Custom Resource.

Fixes VZ-5700

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/update/opensearch/opensearch_installargs_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+package opensearch
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = t.Describe("Update opensearch", Label("f:platform-lcm.update"), func() {
+
+	// GIVEN a VZ custom resource in dev or prod profile,
```
Comment (anders-swanson) [inline tests/e2e/update/opensearch/opensearch_installargs_test.go:12]: please remove commented out test

File: tests/e2e/update/opensearch/opensearch_update.go [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+	"github.com/verrazzano/verrazzano/tests/e2e/pkg"
+	"github.com/verrazzano/verrazzano/tests/e2e/update"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"strconv"
+	"time"
+)
+
+const (
+	//masterNodeName  = "system-es-master"
```
Comment (anders-swanson) [inline tests/e2e/update/opensearch/opensearch_update.go:19]: remove commented values unless they have some significance

**Comment by anders-swanson** [comment]
Because this is changing the topology, will there be a conflict with the opensearch topology test?

File: tests/e2e/update/opensearch/opensearch_installargs_test.go (hunk 1 above)
Comment (anajosep) [reply tests/e2e/update/opensearch/opensearch_installargs_test.go]: Removed all commented tests related to `install-args`

File: tests/e2e/update/opensearch/opensearch_update.go (hunk 2 above)
Comment (anajosep) [reply tests/e2e/update/opensearch/opensearch_update.go]: Remove all commented lines of code.

**Comment by anajosep** [comment]
> Because this is changing the topology, will there be a conflict with the opensearch topology test? I am also not sure if we need this test with opensearch toplogy test as well. similar test suite

There is no impact to the topology test. I have moved the tests to a separate dynamic config update Jenkinsfile where such impacting tests are run. So there should not be any impact. The difference between topology tests and this one is that this one edits the VZ CR and the topology edits the VMI directly.

**Review by anders-swanson — APPROVED** [review]
LGTM; note that opensearch topology test had some resource starvation when creating larger clusters. not sure if you will see this on your test - if intermittent failures pop up, know that it could be a resource problem since you have 10 opensearch nodes on the cluster.

**Pull request merged**